airflow/executors/kubernetes_executor.py

```diff
@@ -368,17 +368,17 @@ def delete_pod(self, pod_id: str, namespace: str) -> None:
             if e.status != 404:
                 raise
 
-    def patch_pod_executor_done(self, *, pod_id: str, namespace: str):
+    def patch_pod_executor_done(self, *, pod_name: str, namespace: str):
```

Comment:
This one isn't in a release yet, so it's fair game.

Reply:
I actually moved this specific change out into #29147, so it can be easily cherry-picked to 2.5.2.